other/materials_designer/passivate_slab.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    if not slab.metadata or slab.metadata["build"]["configuration"]["type"] != SlabConfiguration.__name__:
</pre><p>create slab if not</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>